data/ALIPHATICHYD.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Fuad ALIPHATIC",
+  "bio": "Software Engineer | Product Manager | Public Speaker"
```

```suggestion
  "bio": "Software Engineer | Product Manager | Public Speaker",
```
Please add a comma here